### DISCUSSION TITLE
How to add this to webpack config?

### DISCUSSION THREAD

**Discussion by kamrava** [quasarframework/quasar#10339] [General - Components / Directives / etc] [answered]
I want to add this:

```
exprContextCritical: false
```
to webpack config in quasar.conf.js file.
You can see webpack configuration [here](https://webpack.js.org/configuration/module/#module-contexts).
Do you have any idea?

**Comment by MilosPaunovic** [comment]
Check this out:
https://quasar.dev/quasar-cli/handling-webpack

**Reply by kamrava** [reply]
Thanks but I have seen that page before. I got confused where to put ```exprContextCritical: false```

**Comment by kamrava** [comment, marked as answer]
I found the answer:
```
extendWebpack(cfg) {
        cfg.module.exprContextCritical = false,
        ...
}
```
